pipeline.py

```diff
@@ -88,12 +103,27 @@ def validate_dataset(path_to_validate):
     """
     Validates folder with assets
     """
-    pass
+
+    if not path_to_validate.is_dir():
+        raise NotADirectoryError
+
+    if not any(Path(path_to_validate).iterdir()):
+        raise EmptyDirectoryError
+
+    file_formats = [".json", ".txt", ".pdf"]
+    files = path_to_validate.iterdir()
+    for i, file in enumerate(files, 1):
+        if str(i) != str(file.name)[0]:
+            raise InconsistentDatasetError
+        if file.suffix not in file_formats:
```

Comment:
checks should be much much much more complex. read the docs carefully

Reply:
Okay